build-scripts/compile_all.py

```diff
@@ -121,6 +134,40 @@ def find_existing_rules(project_root):
     return rules
 
 
+def get_relevant_benchmarks(env_yaml, product_yaml):
+    benchmark_paths = get_all_content_directories(env_yaml, product_yaml)
+    out = set()
+    for benchmark in BENCHMARKS:
+        for path in benchmark_paths:
+            if benchmark in path:
+                out.add(benchmark)
+    return out
+
+
+def get_minimal_profiles_by_id(rules, variables):
+    out = {}
+    for rule in rules:
+        data = {
+              'documentation_complete': True,
+              'variables': variables,
```

Comment:
```suggestion
              'variables': variables,
              'hidden': True,
```

I suggest creating a hidden profile so that this fake profile won't appear in the built SCAP data stream.